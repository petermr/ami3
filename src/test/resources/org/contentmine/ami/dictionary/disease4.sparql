SELECT ?wikidata ?wikidataLabel ?wikidataAltLabel ?wikidataDescription ?wikipedia WHERE {
    ?wikidata wdt:P31 wd:Q12136 .
	SERVICE wikibase:label {
		bd:serviceParam wikibase:language "en"
	}
	OPTIONAL {
		?wikipedia schema:about ?wikidata .
		?wikipedia schema:inLanguage "en" . 
		?wikipedia schema:isPartOf <https://en.wikipedia.org/> .
	} 
}
LIMIT 4
