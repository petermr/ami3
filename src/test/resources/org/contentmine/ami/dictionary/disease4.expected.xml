<?xml version="1.0" encoding="UTF-8"?>
<dictionary title="disease4">
 <entry description="injury caused by a bite from a snake" name="snakebite" term="snakebite" wikidata="Q68854" wikidataAltLabel="snake bite, snake bites, snake envenomation, snake envenoming" wikipedia="Snakebite">
  <synonym>snake bite</synonym>
  <synonym>snake bites</synonym>
  <synonym>snake envenomation</synonym>
  <synonym>snake envenoming</synonym>
 </entry>
 <entry description="Human disease: infectious thrombophlebitis of the internal jugular vein" name="Lemierre's syndrome" term="Lemierre's syndrome" wikidata="Q72000" wikidataAltLabel="Lemierre syndrome, acute sore throat, human necrobacillosis, postanginal sepsis" wikipedia="Lemierre%27s_syndrome">
  <synonym>Lemierre syndrome</synonym>
  <synonym>acute sore throat</synonym>
  <synonym>human necrobacillosis</synonym>
  <synonym>postanginal sepsis</synonym>
 </entry>
 <entry description="endocarditis that results from the deposition of small sterile vegetations on valve leaflets" name="marantic endocarditis" term="marantic endocarditis" wikidata="Q73518" wikidataAltLabel="non-bacterial thrombotic endocarditis, non-infective endocarditis, nonbacterial thrombotic endocarditis" wikipedia="Nonbacterial_thrombotic_endocarditis">
  <synonym>non-bacterial thrombotic endocarditis</synonym>
  <synonym>non-infective endocarditis</synonym>
  <synonym>nonbacterial thrombotic endocarditis</synonym>
 </entry>
 <entry description="type of inflammatory bowel disease" name="Crohn's disease" term="Crohn's disease" wikidata="Q1472" wikidataAltLabel="Crohn disease, Crohn, Crohn's disease of colon, Crohn's disease of colon (disorder), Crohn's disease of large bowel, Granulomatous Colitis, Pediatric Crohn's disease, regional colitis, regional enteritis, regional enteritis of small intestine with large intestine, regional enteritis of the large bowel, regional Ileitis, regional ileocolitis" wikipedia="Crohn%27s_disease">
  <synonym>Crohn disease</synonym>
  <synonym>Crohn</synonym>
  <synonym>Crohn's disease of colon</synonym>
  <synonym>Crohn's disease of colon (disorder)</synonym>
  <synonym>Crohn's disease of large bowel</synonym>
  <synonym>Granulomatous Colitis</synonym>
  <synonym>Pediatric Crohn's disease</synonym>
  <synonym>regional colitis</synonym>
  <synonym>regional enteritis</synonym>
  <synonym>regional enteritis of small intestine with large intestine</synonym>
  <synonym>regional enteritis of the large bowel</synonym>
  <synonym>regional Ileitis</synonym>
  <synonym>regional ileocolitis</synonym>
 </entry>
</dictionary>
